polaris-react/src/components/Avatar/Avatar.scss

```diff
@@ -82,34 +104,114 @@
   color: rgba(61, 40, 0, 1);
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   background: rgba(255, 201, 107, 1);
+
+  #{$se23} & {
+    background: var(--p-color-avatar-background-yellow-experimental);
+  }
+
+  svg {
+    #{$se23} & {
+      color: var(--p-color-avatar-icon-yellow-experimental);
+    }
+  }
+
+  text {
+    #{$se23} & {
+      color: var(--p-color-avatar-text-yellow-experimental);
+    }
+  }
```

Comment:
```suggestion
  #{$se23} & svg,
  #{$se23} & text {
    color: var(--p-color-avatar-icon-yellow-experimental);
  }
```

Light suggestion

Reply:
These are applying two different colors to icons vs. text so I don't think they can be combined. 

- `--p-color-avatar-icon-yellow-experimental`
- `--p-color-avatar-text-yellow-experimental`

Reply:
Ah yes! Missed that subtle nuance 👍